toolkit/tools/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/microsoft/azurelinux/toolkit/tools
 
-go 1.20
+go 1.21
```

Comment:
The PR check seems to be failing as it expects go1.20. Maybe we should fix that as well while we are here.

Reply:
Heh, hoisted by my own petard! https://github.com/microsoft/azurelinux/pull/6329/files#diff-34ce0ad126a3f6ca4a539ed80076f4d2df76eb3622f49f759236d1108e3040c7R41

That said, I put that there for a reason...